src/t_hash.c

```diff
@@ -2746,85 +2770,50 @@ void hgetdelCommand(client *c) {
  */
 void hgetexCommand(client *c) {
     int expired = 0, deleted = 0, updated = 0;
-    int num_fields_pos = 3, cond = 0;
-    long num_fields;
-    int64_t oldlen = 0, newlen = -1;
+    int parse_flags = 0, expire_time_pos = -1, first_field_pos = -1, field_count = -1;
```

Comment:
what about keeping the original variable names? in that case we can avoid some changes.